tests/test_converters.py

```diff
@@ -0,0 +1,87 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# Copyright (c) 2020 Scipp contributors (https://github.com/scipp)
+# @file
+# @author Matthew Andrew
+
+import pytest
+from scwidgets.converters import (filepath_converter, scope_converter,
+                                  typed_scope_converter,
+                                  string_allowed_values_converter)
+
+
+@pytest.fixture
+def temp_filepath(tmp_path, scope="module"):
+    tmp_file = tmp_path / "test_file.txt"
+    tmp_file.write_text('content')
+
+    return tmp_file
```

Comment:
Use `tempfile`, something like done in https://github.com/scipp/scipp/blob/61596b8041c789f659e458edf5b1d4f86e7166b6/python/tests/io/test_hdf5.py#L12.